woocommerce/woocommerce-ios
# Shipping Label Details > Ship To: update "House" icon to outlined house icon

Part of #3334 (Update "House" icon to outlined house icon)

## Why

Adding a new icon to [Gridicons](https://github.com/Automattic/gridicons) is a big undertaking (requires review from Gridicons maintainers) (ref CGPNUU63E/p1607084416034700-slack), so this PR just added the house PDF from design.

## Changes

- Exported `icon-house-outlined` PDF from design (I updated the stroke color to black in a copied design file, otherwise the color is slightly off)
- Added `UIImage.houseOutlinedImage` with a test case
- In `ShippingLabelDetailsViewController`, updated "Ship To" row to use `houseOutlinedImage` image

## Testing

Prerequisites: the site has [WooCommerce Shipping & Tax plugin](https://docs.woocommerce.com/document/woocommerce-shipping-and-tax/) and there is an order with at least one shipping label.

You can set up your store to use a test credit card with instructions in p91TBi-3xD-p2.

- Go to the orders tab
- Tap on an order with at least one shipping label
- After a shipping label package card appears, tap "View Shipment Details" --> the "Ship to" row should show the house outlined icon instead of solid icon

## Example screenshots

before | after
-- | --
![Simulator Screen Shot - iPhone 11 - 2020-12-22 at 18 37 39](https://user-images.githubusercontent.com/1945542/102880985-252e5800-4487-11eb-8d0c-f640f742ca83.png) | ![Simulator Screen Shot - iPhone 11 - 2020-12-22 at 18 53 16](https://user-images.githubusercontent.com/1945542/102880999-2a8ba280-4487-11eb-927c-41d3356852b1.png)
![Simulator Screen Shot - iPhone 11 - 2020-12-22 at 18 37 51](https://user-images.githubusercontent.com/1945542/102880991-28c1df00-4487-11eb-8248-cdc9aaec0dd1.png) | ![Simulator Screen Shot - iPhone 11 - 2020-12-22 at 18 53 09](https://user-images.githubusercontent.com/1945542/102880997-29f30c00-4487-11eb-8f2a-0a7108755631.png)


Update release notes:
